migration/src/main/java/com/github/gfx/android/orma/core/DefaultDatabase.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2015 FUJI Goro (gfx).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.github.gfx.android.orma.core;
+
+import android.annotation.TargetApi;
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.DatabaseUtils;
+import android.database.sqlite.SQLiteDatabase;
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+public class DefaultDatabase implements Database {
+
+    private final SQLiteDatabase database;
+
+    public DefaultDatabase(SQLiteDatabase database) {
+        this.database = database;
+    }
```

Comment:
This constructor is used in some tests now.
When the tests stop using this constructor, we can change the visibility to private.